main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -501,7 +500,9 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
   /**
     * Builds a jar package for the project.
     */
-  def buildJar(): Validation[Unit, BootstrapError] = {
+  def buildJar(flix: Flix): Validation[Unit, BootstrapError] = {
```

Comment:
I think there was a point in these not requiring an Flix object.

In other words, they are independent of the specific flix compiler instance.

I think we should just pass the formatter implicitly.